setup.cfg

```diff
@@ -79,7 +79,7 @@ doc = sphinx>=4.4.0
     sphinxcontrib-apidoc>=0.3.0
 extras = pynput>=1.7.4  # Used by HumanKeyboardAgent
 gym = moviepy == 1.0.3
-remote_agent = grpcio==1.32.0
+remote-agent = grpcio==1.32.0
```

Comment:
Is this change necessary? The direction we are going is to have all of the extras remove dash.

Reply:
Reverted accidental change.

Reply:
On second review, this is the cause of all of the extras having the `-` -> `_` conversion.

Reply:
Shall we simply convert all extras to use `_`, to avoid confusion and to avoid auto name changes?

Reply:
I think we can do that because the next release is a major version change.

Reply:
For now we should use `remote-agent` and then have an issue to convert the rest.

Reply:
Conversion of `-` to `_` in `options.extras_require` is done in #1756 .